src/index.html

```diff
@@ -15,13 +22,330 @@
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
+          />
+          <h2 class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__cost">
+            <p class="card__price">Price:</p>
+            <p class="card__total">$2,199</p>
+          </div>
+
+          <button class="card__button">Buy</button>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
+          />
+          <h2 class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__cost">
+            <p class="card__price">Price:</p>
+            <p class="card__total">$2,199</p>
+          </div>
+
+          <button class="card__button">Buy</button>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
+          />
+          <h2 class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__cost">
+            <p class="card__price">Price:</p>
+            <p class="card__total">$2,199</p>
+          </div>
+
+          <button class="card__button">Buy</button>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
```

Comment:
There is an extra space at the beginning of the `alt` attribute value. Consider removing it to maintain consistency and avoid potential issues with screen readers.